pkg/backplane/ca/ca.go

```diff
@@ -40,10 +40,10 @@ const (
 
 var (
 	pkixName = pkix.Name{
-		Organization:       []string{"Heptio"},
-		OrganizationalUnit: []string{"sonobuoy"},
+		Organization:       []string{"VMware"},
+		OrganizationalUnit: []string{"Sonobuoy"},
 		Country:            []string{"USA"},
-		Locality:           []string{"Seattle"},
+		Locality:           []string{"Palo Alto, CA"},
```

Comment:
Calling this section out just for importance; I believe it is just for generating the certs between the aggregator/worker so the details don't really matter.